vulkan/vulkan_utilities.cc

```diff
@@ -33,8 +33,8 @@ static std::vector<std::string> InstanceOrDeviceLayersToEnable(
   const std::vector<std::string> candidates = {
       "VK_LAYER_GOOGLE_threading",      "VK_LAYER_LUNARG_parameter_validation",
       "VK_LAYER_LUNARG_object_tracker", "VK_LAYER_LUNARG_core_validation",
-      "VK_LAYER_LUNARG_device_limits",  "VK_LAYER_LUNARG_image",
-      "VK_LAYER_LUNARG_swapchain",      "VK_LAYER_GOOGLE_unique_objects"};
+      "VK_LAYER_LUNARG_device_limits",  "VK_LAYER_LUNARG_swapchain",
```

Comment:
I'd put this behind `OS_FUCHSIA` or something because the loader is still present on Android. But I don't have a strong opinion about it.